test/fixtures/linode_instances_123_firewalls.json

```diff
@@ -0,0 +1,50 @@
+[{
```

Comment:
It looks like this fixture might not follow the API's response structure: https://www.linode.com/docs/api/linode-instances/#firewalls-list__response-samples

At the top-level the API returns a `data` field containing the firewalls and various other pagination data, so we'll probably want to reflect this in the fixture 👍

Reply:
Nice catch! Looks like this was the case for nodebalancers and volumes as well. Fixed the fixtures for all three.